src/base/RandomUuid.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 1996-2022 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef SQUID_SRC_BASE_RANDOM_UUID_H
+#define SQUID_SRC_BASE_RANDOM_UUID_H
+
+#include <array>
+#include <iosfwd>
+
+/// 128-bit Universally Unique IDentifier (UUID), version 4 (variant 1) as
+/// defined by RFC 4122. These UUIDs are generated from pseudo-random numbers.
+class RandomUuid
+{
+public:
+    using Serialized = std::array<char, 128/8>;
```

Comment:
```suggestion
    /// UUID representation independent of machine byte-order architecture
    using Serialized = std::array<char, 128/8>;
```

Please adjust the RandomUuid implementation to make serialized UUIDs portable across machines with different byte order.

Reply:
Done.